src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -382,7 +384,7 @@ MockExpectedCall_c* withConstPointerParameters_c(const char* name, const void* v
 
 MockExpectedCall_c* withFunctionPointerParameters_c(const char* name, void (*value)())
 {
-    expectedCall = &expectedCall->withParameter(name, value);
+    expectedCall = &expectedCall->withParameter(name, (Cpp_Func*)value);
```

Comment:
This can be done; however, since I may not add a comment to explain what these apparently redundant casts are for; some bright mind will remove it in the future, at which point, since Travis will be fine (and nobody is going to complain on the Cl2000 side) the Cl2000 build will silently break :P.